fintech-examples/fintech-ui/Dockerfile

```diff
@@ -48,14 +48,14 @@ COPY ./nginx.conf.erb /etc/nginx/conf.d/default.conf.erb
 # copy entry.sh file
 COPY ./entry.sh /opt/entry.sh
 
-# set permissions
-RUN chgrp -R root /var/cache/nginx && \
-    find /var/cache/nginx -type d -exec chmod 775 {} \; && \
-    find /var/cache/nginx -type f -exec chmod 664 {} \; && \
-    chmod 775 /var/run && \
+# support running as arbitrary user which belogs to the root group
+RUN chmod g+rwx /var/cache/nginx /var/run /var/log/nginx && \
     chmod 775 /opt/entry.sh && \
     chmod -R 777 /etc/nginx/conf.d
 
+# comment user directive as master process is run as user in OpenShift anyhow
+RUN sed -i.bak 's/^user/#user/' /etc/nginx/nginx.conf
```

Comment:
Theoretically Docker-image shall be run everywhere, regardless if it is openshift or not.
So I'd rather recommend to align userId between Dockerfile and nginx.conf and forget about this problem.

Reply:
True, aligning the user makes sense. We should do this in a separate PR. I will revert these two changes in this PR.